web/src/routes/App.component.tsx

```diff
@@ -55,6 +55,7 @@ export type AppStateProps = {
   whoami: WireRecord<Profile>
   hasFetchedForWhoami: boolean
   navigationPreference: 'mouse' | 'trackpad'
+  keyboardNavigationPreference: 'coordinates' | 'modal'
```

Comment:
actually, 1 more change request. can you refactor instances of the 'coordinates' | 'modal' and even also 'mouse' | 'trackpad' into type aliases that get shared around and imported?